lib/Common/Memory/Recycler.cpp

```diff
@@ -5968,7 +5968,7 @@ Recycler::ThreadProc()
     }
 #endif
 
-#ifdef ENABLE_JS_ETW
+#if defined(ENABLE_JS_ETW) && ! defined(ENABLE_JS_LTTNG)
```

Comment:
Why?

Reply:
Because LTTng doesn't have any concept of `EventActivityIdControl`

Reply:
Perhaps call that out as the reason why, here?